defaults/test_default.yaml

```diff
@@ -63,3 +63,8 @@ monitor_swap_size: 8192 # Size of swap file for monitor node in bytes 8192 * 1MB
 compaction_strategy: 'SizeTieredCompactionStrategy'
 
 use_preinstalled_scylla: false
+
+alternator_enforce_authorization: false
+alternator_access_key_id: ''
```

Comment:
it may sound not clear to have these 2 different params - access-key and secret-access-key.
the names may not imply how they both are related. or is there documentation of this in other place?

Reply:
it's a common AWS name for those:
see:
https://github.com/scylladb/scylla/blob/master/docs/alternator/alternator.md#api-server
https://github.com/scylladb/scylla/blob/master/test/alternator/test_authorization.py